src/windows/BarcodeScannerProxy.js

```diff
@@ -243,7 +246,19 @@ module.exports = {
             // rotate the preview video
             var videoEncodingProperties = capture.videoDeviceController.getMediaStreamProperties(Windows.Media.Capture.MediaStreamType.videoPreview);
             videoEncodingProperties.properties.insert(ROTATION_KEY, rotDegree);
-            return capture.videoDeviceController.setMediaStreamPropertiesAsync(Windows.Media.Capture.MediaStreamType.videoPreview, videoEncodingProperties);
+
+            try {
+                return capture.videoDeviceController.setMediaStreamPropertiesAsync(Windows.Media.Capture.MediaStreamType.videoPreview, videoEncodingProperties);
+            } catch (e) {
+                if (e.number !== PHOTO_SEQUENCE_IS_IN_PROGRESS) {
+                    throw e;
```

Comment:
> ok to throw an exception as it will be caught in the topmost promise

Yeah, i noticed that. It is more like a good pattern to follow - function should return both success and error consistently - either in sync or async way.